src/AppBundle/Controller/AdherentController.php

```diff
@@ -13,8 +15,22 @@ class AdherentController extends Controller
     /**
      * @Route("/mon-profil", name="app_adherent_profile")
      */
-    public function profileAction()
+    public function profileAction(): Response
     {
         return $this->render('adherent/profile.html.twig');
     }
+
+    /**
+     * This action enables a new user to pin his/her interests.
+     *
+     * TODO find a better path
+     *
+     * @Route("/interets", name="app_adherent_pin_interests")
```

Comment:
`centres-interets`, but if you put it under `/espace-adherent` won't there be issues with the security?